netdata/Modules/ServerDetail/Components/AlarmsListView.swift

```diff
@@ -35,8 +35,13 @@ struct AlarmsListView: View {
                     }
                 }
             }
+            .navigationTitle(Text("Active Alarms")).navigationBarTitleDisplayMode(.inline)
+            /*.toolbar {
```

Comment:
Is this not required?

Reply:
you bean the toolbar? Thats the iOS 14 way of making toolbars the only problem is when i tried using it with a sheet to present a view it doesn't apply the button styles to the buttons it just presents them as normal/standard button/text 
if you use navigationBarItems instead it works as expected 🤪
it might be a bug with iOS